### ISSUE TITLE
force-quite: Force Quite applet does not work ! Please fix.

### ISSUE THREAD

**Issue by Nokia808** [linuxmint/cinnamon-spices-applets#69]
Hi. Yesterday I installed this icon & it does not work at all. I opened a window for a game as a test, then click on it but xkill not activated & mouse pointer not converted to x & when I use mouse pointer to close window of game or any other windows it did not work at all.

Your kind fix please.

I'm on Fedora 24 X64 Cinnamon version

Best.

**Comment by NikoKrause** [comment]
Have you tried Force Quit II?

**Comment by Nokia808** [comment]
I try Force Quit II. It has other form of bug:

Force Quite II work for only one time !! After that if you try to use it again it will never work ! Even after restart PC not work. Moreover, there is very strange thing: I try to uninstall it & reinstall it but not worked also, but when I uninstall it then restart PC then reinstall it it will work but also for one time !!! So it is useless. I will post this in new issue about Force Quite II.

**Comment by NikoKrause** [comment]
You don't need to open another issue for that. This one suffices.

I tried to look into it. It works for me, when I use the touchpad mouse button. But not, when I use the touchpad itself.
I get the error:
`unable to grab cursor`

It seems to be an error with xkill itself. Don't know how to fix it.

**Comment by NikoKrause** [comment]
@mohammad-sn

**Comment by mohammad-sn** [comment]
It turns out xkill should get called only after releasing the mouse button. Calling xkill on button release event fixes it. I made a PR to fix it.

**Comment by NikoKrause** [comment]
@mohammad-sn

**Comment by NikoKrause** [comment]
@mohammad-sn 
Thanks for fixing this issue. I tested it and it works for me.
There are two "Force Quit" applets.
Since you fixed the original version, can we delete "Force Quit II"?

**Comment by mohammad-sn** [comment]
@NikoKrause 
Thanks.
